stores/rocksdb/src/commonMain/kotlin/kotlinx/document/database/rocksdb/RocksdbDataStore.kt

```diff
@@ -1,23 +1,66 @@
 package kotlinx.document.database.rocksdb
 
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.IO
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 import kotlinx.document.database.DataStore
 import kotlinx.document.database.PersistentMap
 import maryk.rocksdb.Options
 import maryk.rocksdb.RocksDB
+import maryk.rocksdb.WriteOptions
 import maryk.rocksdb.openRocksDB
 import maryk.rocksdb.use
+import kotlin.coroutines.CoroutineContext
+
+public class RocksdbDataStore(
+    internal val delegate: RocksDB,
+    override val configuration: DataStore.Configuration,
+) : DataStore, CoroutineScope {
+    override val coroutineContext: CoroutineContext by lazy { SupervisorJob() + Dispatchers.IO }
+
+    init {
+        when (val policy = configuration.commitPolicy) {
```

Comment:
Use an `if` not a `when`. The `when` forces you to create all branches, the `if` does not.